src/bindings/python/src/pyopenvino/core/compiled_model.cpp

```diff
@@ -47,7 +48,10 @@ void regclass_CompiledModel(py::module m) {
             auto request = self.create_infer_request();
             // Update inputs if there are any
             Common::set_request_tensors(request, inputs);
-            request.infer();
+            {
+                py::gil_scoped_release release;
```

Comment:
No info on releasing GIL.

Moreover please connect it with this statement: "Blocks all methods of CompiledModel while the request is running".